Automattic/wp-calypso
# Jetpack Partner Portal: Introduce payment method page and conditionally add sidebar menu item

**Note:** This PR is a work in progress, and will be broken down into smaller bits before shipping.

#### Changes proposed in this Pull Request

* Introduces the `jetpack/partner-portal-payment` feature flag
* Adds payment method page (feature-flagged) to Jetpack Partner Portal
* Adds sidebar link (feature-flagged) to the sidebar in Jetpack Partner Portal
* Rearranges the sidebar icons per i2 designs

#### Testing instructions

* Load up the partner portal in `jetpack-cloud-development`.
* Ensure that the new sidebar menu item "Payment Method" is present.
* Ensure that when clicking the sidebar menu item you are directed to a new page.
* Ensure you see a list of your account's credit cards (currently inherits WPCOMs stripe account credit cards).
* Ensure you see the "Add credit card form" when visiting that page (You can't actually submit a card yet).
* Observe javascript console for errors and/or warnings.



Related to #
